server/src/main/java/io/spine/server/entity/Transaction.java

```diff
@@ -445,6 +439,61 @@ public void setDeleted(boolean deleted) {
     }
 
     /**
+     * Retrieves the {@link VersionAdvancementStrategy} of current {@code Transaction}.
+     *
+     * <p>The value should be constant along all the instances of a certain (runtime) type.
+     *
+     * <p>The default value is {@link VersionAdvancementStrategy#INJECT INJECT}.
+     */
+    protected VersionAdvancementStrategy versioningStrategy() {
+        return INJECT;
+    }
+
+    /**
+     * The strategy of advancing the version of a {@code Transaction} upon the given {@link Phase}.
```

Comment:
`The strategy of versioning of entities during a phase of a certain transaction.` (just add proper Javadoc links etc).